mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -74,6 +74,48 @@ macro_rules! event {
     }
 }
 
+#[derive(Debug, Clone, Default)]
```

Comment:
I don't think empty string is the right default for most of these, so should manually implement `Default`. But actually, I don't think we want this to implement `Default`, because there's no sensible default value for `signing_region`?

Reply:
I think we can have empty string here. If we find empty string or '*' (as in the case of Multi Region Access Points), we can set it to `EndpointConfig` region.